ERROR - 2023-04-03 00:22:51 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'usuarios_umc' in C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'usuarios_umc', 3306, '', 0)
#1 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\danai\xampp\htdocs\Proyecto-Uni\app\Models\UsuarioModel.php(14): CodeIgniter\Model->insert(Array)
#8 C:\Users\danai\xampp\htdocs\Proyecto-Uni\app\Controllers\register.php(23): App\Models\UsuarioModel->registrarUsuario(Array)
#9 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\CodeIgniter.php(934): App\Controllers\register->registrar()
#10 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\register))
#11 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\Users\danai\xampp\htdocs\Proyecto-Uni\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'usuarios_umc' in C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\Users\danai\xampp\htdocs\Proyecto-Uni\app\Models\UsuarioModel.php(14): CodeIgniter\Model->insert(Array)
#7 C:\Users\danai\xampp\htdocs\Proyecto-Uni\app\Controllers\register.php(23): App\Models\UsuarioModel->registrarUsuario(Array)
#8 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\CodeIgniter.php(934): App\Controllers\register->registrar()
#9 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\register))
#10 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\danai\xampp\htdocs\Proyecto-Uni\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-04-03 00:22:51 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'usuarios_umc'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `usuario`, `cedula`, `password`) VALUES (:nombre:, :apellido:, :usuario:, :cedula:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Models\UsuarioModel.php(14): CodeIgniter\Model->insert([...])
 7 APPPATH\Controllers\register.php(23): App\Models\UsuarioModel->registrarUsuario([...])
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\register->registrar()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\register))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-03 03:14:39 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'usuarios_umc' in C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'usuarios_umc', 3306, '', 0)
#1 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\danai\xampp\htdocs\Proyecto-Uni\app\Models\UsuarioModel.php(14): CodeIgniter\Model->insert(Array)
#8 C:\Users\danai\xampp\htdocs\Proyecto-Uni\app\Controllers\register.php(23): App\Models\UsuarioModel->registrarUsuario(Array)
#9 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\CodeIgniter.php(934): App\Controllers\register->registrar()
#10 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\register))
#11 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\Users\danai\xampp\htdocs\Proyecto-Uni\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'usuarios_umc' in C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\Users\danai\xampp\htdocs\Proyecto-Uni\app\Models\UsuarioModel.php(14): CodeIgniter\Model->insert(Array)
#7 C:\Users\danai\xampp\htdocs\Proyecto-Uni\app\Controllers\register.php(23): App\Models\UsuarioModel->registrarUsuario(Array)
#8 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\CodeIgniter.php(934): App\Controllers\register->registrar()
#9 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\register))
#10 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\danai\xampp\htdocs\Proyecto-Uni\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-04-03 03:14:39 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'usuarios_umc'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `usuario`, `cedula`, `password`) VALUES (:nombre:, :apellido:, :usuario:, :cedula:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Models\UsuarioModel.php(14): CodeIgniter\Model->insert([...])
 7 APPPATH\Controllers\register.php(23): App\Models\UsuarioModel->registrarUsuario([...])
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\register->registrar()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\register))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-03 03:41:28 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'usuarios_umc' in C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'usuarios_umc', 3306, '', 0)
#1 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\danai\xampp\htdocs\Proyecto-Uni\app\Models\UsuarioModel.php(14): CodeIgniter\Model->insert(Array)
#8 C:\Users\danai\xampp\htdocs\Proyecto-Uni\app\Controllers\register.php(23): App\Models\UsuarioModel->registrarUsuario(Array)
#9 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\CodeIgniter.php(934): App\Controllers\register->registrar()
#10 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\register))
#11 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\Users\danai\xampp\htdocs\Proyecto-Uni\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'usuarios_umc' in C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\Users\danai\xampp\htdocs\Proyecto-Uni\app\Models\UsuarioModel.php(14): CodeIgniter\Model->insert(Array)
#7 C:\Users\danai\xampp\htdocs\Proyecto-Uni\app\Controllers\register.php(23): App\Models\UsuarioModel->registrarUsuario(Array)
#8 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\CodeIgniter.php(934): App\Controllers\register->registrar()
#9 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\register))
#10 C:\Users\danai\xampp\htdocs\Proyecto-Uni\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\danai\xampp\htdocs\Proyecto-Uni\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-04-03 03:41:28 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'usuarios_umc'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `usuario`, `cedula`, `password`) VALUES (:nombre:, :apellido:, :usuario:, :cedula:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Models\UsuarioModel.php(14): CodeIgniter\Model->insert([...])
 7 APPPATH\Controllers\register.php(23): App\Models\UsuarioModel->registrarUsuario([...])
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\register->registrar()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\register))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-04-03 03:59:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
